modules/manage/pages/kubernetes/manage-connectors.adoc

```diff
@@ -0,0 +1,111 @@
+= Create and Manage Connectors in Kubernetes
+:description: Learn how to create and manage connectors using Redpanda Console or the Kafka Connect REST API.
+
+When you have Redpanda Connectors deployed, you can create and manage connectors using Redpanda Console or the Kafka Connect REST API.
+
+== Prerequisites
+
+- xref:deploy:deployment-option/self-hosted/kubernetes/kubernetes-deploy.adoc[Deploy a Redpanda cluster with Redpanda Console].
+- xref:deploy:deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc[Deploy Redpanda Connectors].
+
+== Manage connectors in Redpanda Console
+
+By default, Redpanda Console is deployed with a ClusterIP Service. To access Redpanda Console, you can use the `kubectl port-forward` command to forward one of your local ports to the Pod.
+
+NOTE: The `kubectl port-forward` command is a development tool. For exposing services to external traffic in a more permanent and controlled manner, use Kubernetes Services such as LoadBalancer or NodePort.
+
+. Expose Redpanda Console to your localhost:
++
+[,bash]
+----
+kubectl --namespace <namespace> port-forward svc/redpanda-console 8080:8080
+----
++
+This command actively runs in the command-line window. To execute other commands while the command is running, open another command-line window.
+
+. Open Redpanda Console on http://localhost:8080.
+
+You can create and manage connectors by clicking *Connectors* in the left navigation menu.
```

Comment:
I'd leave out the location, as these things have a way of moving around. Confluent had a rule to never describe the location of an element in the UI. Not saying we do, but if/when it ever changes, then you have to update it.